.circleci/config.yml

```diff
@@ -2,68 +2,32 @@
 version: 2.1
 orbs:
   commitlint: conventional-changelog/commitlint@1.0.0
-  
-executors:
-  default-executor:
-    docker:
-      - image: "cimg/node:10.22.0"
-    working_directory: ~/project
-    resource_class: medium
-
-commands:
-  restore_test_cache:
-    steps:
-      - restore_cache:
-          keys:
-            - v1-deps-{{ checksum "package-lock.json" }}
-            - v1-deps-
-  save_test_cache:
-    steps:
-      - save_cache:
-          key: v1-deps-{{ checksum "package-lock.json" }}
-          paths:
-            - node_modules
```

Comment:
These cache commands are no longer necessary: the `node/install-packages` command does its own very fancy caching.